### PR TITLE
Adding session for requests, so it works with dns errors.

### PR DESCRIPTION
**Pull request by caiocacastro**
Look at the code and give your thoughts

### CODE REVIEW

File: service_load_data_elasticsearch/settings.py [hunk 1]
```diff
@@ -172,6 +172,9 @@
 ELASTICSEARCH_URL = os.getenv(
     "ELASTICSEARCH_URL", "http://es.xskylab.com:9200")
 
+# Variable controlling SLL validation for es requests
+VERIFY_SSL = bool(distutils.util.strtobool(os.getenv("VERIFY_SSL", False)))
```
Comment (dagnaldo) [inline service_load_data_elasticsearch/settings.py]: Default `True`

File: service_load_data_elasticsearch/settings.py [hunk 2]
```diff
@@ -172,6 +172,9 @@
...
+
 # Increase max size for uploaded file
 # https://docs.djangoproject.com/en/3.2/ref/settings/#file-upload-max-memory-size
-FILE_UPLOAD_MAX_MEMORY_SIZE = 2147483648
+FILE_UPLOAD_MAX_MEMORY_SIZE = 2147483648
```
Comment (dagnaldo) [inline service_load_data_elasticsearch/settings.py]: Remove this config line. please

**Review by dagnaldo — CHANGES_REQUESTED** [review]
config check

File: service_load_data_elasticsearch/settings.py (hunk 1 above)
Comment (caiocacastro) [reply service_load_data_elasticsearch/settings.py]: Done 603aa1d

File: service_load_data_elasticsearch/settings.py (hunk 2 above)
Comment (caiocacastro) [reply service_load_data_elasticsearch/settings.py]: Removed 603aa1d

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**